docker-compose.yaml

```
@@ -173,15 +173,31 @@ services:
 
+  #Service: Redis-pubsub
+  redis-pubsub:
+    image: redis/redis-stack-server:latest
+    ports:
+      - 6379:6379
+    healthcheck:
+    test: ["CMD", "redis-cli", "ping"]
+    interval: 10s
+    timeout: 5s
+    retries: 3
+    restart: always  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)